tests/test_objectives_mayer.py

```diff
@@ -0,0 +1,374 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+import biorbd
+
+from biorbd_optim import (
+    OptimalControlProgram,
+    DynamicsTypeList,
+    DynamicsType,
+    BoundsOption,
+    InitialConditionsOption,
+    ObjectiveOption,
+    Objective,
+    Axe,
+)
```

Comment:
You can use fixture to avoid duplicating code (merge Lagrange, Mayer and Constraints)